api/vertex_test.go

```diff
@@ -40,6 +56,42 @@ func TestHas(t *testing.T) {
 	assert.Equal(t, fmt.Sprintf("%s.V().has('%s','%s')", graphName, key, value), v.String())
```

Comment:
Are you sure that this is correct? Where are key and value defined?

Reply:
they are defined 6 lines above

Reply:
This seemed to be an issue with the diff in Github. Not extremely happy about this WebUI.